handbook/engineering/README.md

```diff
@@ -243,11 +242,11 @@ Immediately after publishing a new release, we close out the associated GitHub i
 
 12. Announce that `main` is unfrozen and the milestone has been closed in #help-engineering.
 
-## Oncall rotation
+## on-call rotation
```

Comment:
Should have capital "O". 


@Sampfluger88 I can see what happened here.  Instead, always manually check each change from a find/replace -- never just do "replace all" unless you're going to immediately go line by line through the diff of a commit.

This was a hard-won lesson for me, and this approach has paid off time and time again.